python/cudf/cudf/core/index.py

```diff
@@ -128,6 +128,25 @@ def __init__(
         """
         pass
 
+    def drop_duplicates(self, keep="first"):
+        """
+        Return Index with duplicate values removed
+
+        Parameters
+        ----------
+        keep : {‘first’, ‘last’, False}, default ‘first’
+            * ‘first’ : Drop duplicates except for the
+                first occurrence.
+            * ‘last’ : Drop duplicates except for the
+                last occurrence.
+            *  False : Drop all duplicates.
+
+        Returns
+        -------
+        deduplicated : Index
+        """
+        return super().drop_duplicates(subset=[self.name], keep=keep)
```

Comment:
Is this different from just calling `super().drop_duplicates`?

Reply:
Nope, changed the call

Reply:
By inheritance we already have `Index.drop_duplicates`, so do we still need this?

Reply:
Because of different function signature for `Series.drop_duplicates` vs `Frame.drop_duplicates` and `Index.drop_duplicates`